kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -1807,6 +1833,38 @@ components:
         - SOURCE
         - SINK
 
+    ConsumerGroupOffsetsReset:
+      type: object
+      properties:
+        topic:
+          type: string
+        resetType:
+          $ref: '#/components/schemas/ConsumerGroupOffsetsResetType'
+        partitions:
+          type: array
+          items:
+            type: integer
+          description: list of target partitions, all partitions will be used if it is not set or empty
+        resetToTimestamp:
+          type: number
+          description: should be set if resetType is TIMESTAMP
+        partitionsOffsets:
```

Comment:
maybe it would be better to pass list of objects with partition int -> offset long?